src/main/java/seedu/address/model/person/Person.java

```diff
@@ -34,13 +35,13 @@ public class Person {
     private final Attendance attendance;
     private final Sessions sessions;
 
-    private final Metadata metadata;
+    private final Optional<Metadata> metadata;
 
     /**
-     * Every field must be present and not null.
+     * Every field must be present and not null exception optional fields
```

Comment:
Technically a Optional<> also shouldn't be null. So this is correct.
```suggestion
     * Every field must be present and not null
```

Reply:
Ohh there's a commit suggestion option that I now just noticed, anyways I've updated it.